manifests/init.pp

```diff
@@ -102,9 +105,9 @@
   $use_sympa           = false,         # postfix_use_sympa
   $postfix_ensure      = 'present',
   $mailx_ensure        = 'present',
-) inherits postfix::params {
-
+) { include ::postfix::params
```

Comment:
Then the standard is to expose `$chroot_daemons` in the main class and make it default to the value from params, and then use the variable from the main class. Cf https://docs.puppet.com/puppet/4.9/lang_classes.html#appendix-smart-parameter-defaults